test/pdfs/.gitignore

```diff
@@ -392,6 +392,7 @@
 !issue6298.pdf
 !issue6889.pdf
 !issue11473.pdf
+!issue13487.pdf
```

Comment:
As mentioned previously, please use a *linked* test-case instead since for copyright reasons we cannot include arbitrary PDF files in the repository; please look at e.g. https://github.com/mozilla/pdf.js/pull/13628/files for an example of how that is done.

Furthermore, please pick a *smaller* PDF document as the test-case (there's a bunch of duplicate issues listed in #11957), since we really don't want to use a 10 MB file unless no other option is available.

Reply:
OK